Bottom: b7a965758dcba2c59c5790362908076e4e36c577
Top:    7da0e0eb0bd7b9fb5903b6c84cd326db44dfd1d1
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-04-06 19:39:17 +0100

Refresh of give-the-hids-control-over-att

---

diff --git a/src/hid/common/hidnogui.c b/src/hid/common/hidnogui.c
index 7c06bdf..cd1c0d2 100644
--- a/src/hid/common/hidnogui.c
+++ b/src/hid/common/hidnogui.c
@@ -480,6 +480,7 @@ apply_default_hid (HID * d, HID * s)
   AD (invalidate_lr);
   AD (invalidate_all);
   AD (notify_crosshair_change);
+  AD (notify_mark_change);
   AD (set_layer);
   AD (make_gc);
   AD (destroy_gc);
